src/transformers/utils/metrics.py

```diff
@@ -1,11 +1,22 @@
 import functools
 import logging
 import time
+from enum import Enum
 from typing import Any, Callable, List, Optional, Tuple, Union
 
 import torch
 
-from ..generation.utils import RequestStatus
+
+class RequestStatus(Enum):
```

Comment:
Why did you duplicate the `RequestStatus` enum @ArthurZucker ?

Reply:
Import issue 🤣